resource.address/spi/src/main/java/org/kaazing/gateway/resource/address/ResourceAddressFactorySpi.java

```diff
@@ -42,7 +45,11 @@
 
 public abstract class ResourceAddressFactorySpi<T extends ResourceAddress> {
 
+    private static final String PREFER_IPV4_STACK_IPV6_ADDRESS_EXCEPTION_FORMATTER = "Option java.net.preferIPv4Stack is set to"
```

Comment:
PREFER_IPV4_STACK_IPV6_ADDRESS_EXCEPTION_FORMATTER is not a formatter correct, just a String.  NO_ADDRESSES_AVAILABLE_FOR_BINDING_FORMATTER is a formatter as it has a &s